3rdparty/CMakeLists.txt

```diff
@@ -496,11 +496,7 @@ if (NBL_BUILD_IMGUI)
 endif()
 
 foreach(trgt IN LISTS NBL_3RDPARTY_TARGETS)
-		if(NBL_DYNAMIC_MSVC_RUNTIME)
-			set_property(TARGET ${trgt} PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
-		else()
-			set_property(TARGET ${trgt} PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
-		endif()
+		set_property(TARGET ${trgt} PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<BOOL:${NBL_COMPILER_DYNAMIC_RUNTIME}>:DLL>")
```

Comment:
AFAIK in Clang and GCC there are options about static linking of libstd-c++

Reply:
yes there [are](<https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-static-libstdc>) and this [property](<https://cmake.org/cmake/help/latest/prop_tgt/MSVC_RUNTIME_LIBRARY.html>) has effect only if MSVC ABI is the target